scripts/README.md

```diff
@@ -2,10 +2,43 @@
 We provide some scripts for reproducing existing algorithms with FederatedScope, which are constantly being updated.
 We greatly appreciate any [contribution](https://federatedscope.io/docs/contributor/) to FederatedScope!
 
+- [Federated Optimization Algorithms](#fed-optimization)
 - [Distribute Mode](#distribute-mode)
 - [Asynchronous Training Strategy](#asynchronous-training-strategy)
 - [Graph Federated Learning](#graph-federated-learning)
 - [Attacks in Federated Learning](#attacks-in-FL)
+- [Differential Privacy in Federated Learning](#dp-in-FL)
+- [Matrix Factorization in Federated Learning](#mf-in-FL)
+
+### Federated Optimization Algorithm
+Users can replace the fedavg algorithm by other federated optimization algorithms.
+In the following we provide some running scripts for FedOpt[1] and FedProx[2] on different dataset.
+
+#### FedOpt
+Run fedopt on different dataset via
+```bash
+# on femnist
+bash optimization_exp_scripts/fedopt_exp_scripts/run_fedopt_femnist.sh
+# on synthetic
+bash optimization_exp_scripts/fedopt_exp_scripts/run_fedopt_lr.sh
+# on shakespeare
+bash optimization_exp_scripts/fedopt_exp_scripts/run_fedopt_shakespeare.sh
+```
+
+#### FedProx
+Run fedprox on different dataset via
+```bash
+# on femnist
+bash optimization_exp_scripts/fedprox_exp_scripts/run_fedprox_femnist.sh
+# on lr
+bash optimization_exp_scripts/fedprox_exp_scripts/run_fedprox_lr.sh
+# on shakespeare
+bash optimization_exp_scripts/fedprox_exp_scripts/run_fedprox_shakespeare.sh
+```
+
+[1] Asad M, Moustafa A, Ito T. "FedOpt: Towards communication efficiency and privacy preservation in federated learning". Applied Sciences, 2020, 10(8): 2864.
```

Comment:
Move all the references to the end of this docs?

Reply:
modified accordingly